File: typescript/src/structural/facade/interfaces/IArticle.ts
export interface Article {
    title: string
    slug: string;
    html?: string;
    content?: string;
    keywords?: string[];
    internalLinks: string[];
    metaDescription: string;
}

export interface Suggestions {
    keywords: string[];
    metaDescription: string[];
    internalLinks: string[];
}

File: c#/Structural/Facade/Interface/IArticle.cs
namespace Interface.IArticle
{
    public interface IArticle
    {
        public string Title { get; set; }
        public string Slug { get; set; }
        public string? Html { get; set; }
        public string? Content { get; set; }
        public List<string> Keywords { get; set; }
        public List<string> MetaDescription { get; set; }
        public List<string> InternalLinks { get; set; }
    }
}